Sources/SwiftLayout/Extension/UIVIew+Layout.swift

```diff
@@ -21,6 +21,11 @@ public extension Layout where Self: UIView {
         ViewLayout(self, sublayout: build())
     }
     
+    func config(_ config: (Self) -> Void) -> some Layout {
+        config(self)
+        return ViewLayout(self, sublayout: EmptyLayout())
```

Comment:
hahahaha this is so what else. I was dumb haha.

Reply:
> hahahaha this is so what else. I was dumb haha.

and its more SwiftLayouty. previous protocol just another utility and didn't have any relation on library.

Reply:
> and its more SwiftLayouty. previous protocol just another utility and didn't have any relation on library.

yes! I modified the code at the point you said